src/sql/plot.py

```diff
@@ -27,21 +27,23 @@ def _summary_stats(con, table, column, with_=None):
     """Compute percentiles and mean for boxplot"""
     template = Template(
         """
-SELECT
-percentile_disc(0.25) WITHIN GROUP (ORDER BY "{{column}}") AS q1,
-percentile_disc(0.50) WITHIN GROUP (ORDER BY "{{column}}") AS med,
-percentile_disc(0.75) WITHIN GROUP (ORDER BY "{{column}}") AS q3,
-AVG("{{column}}") AS mean,
-COUNT(*) AS N
-FROM "{{table}}"
+    SELECT
+    percentile_disc([0.25, 0.50, 0.75]) WITHIN GROUP \
+    (ORDER BY "{{column}}") AS percentiles,
+    AVG("{{column}}") AS mean,
+    COUNT(*) AS N
+    FROM "{{table}}"
 """
     )
+
     query = template.render(table=table, column=column)
 
     if with_:
         query = str(store.render(query, with_=with_))
-
+    query = sql.connection.Connection._transpile_query(query)
     values = con.execute(query).fetchone()
+    # Flatten the ([q1, med, q3], mean, N) to (q1, med, q3, mean, N)
+    values = tuple(values[0]) + (values[1],) + (values[2],)
```

Comment:
We can create a new function in utils e.g `flatten`.
This might be handy in the future/other places (e.g when and if we'll want to add another column/change columns' order we won't need to update values[i]).

Reply:
Oh, I just noticed it's not ready for review. please tag me whenever it's ready.